### PR TITLE
Add openrice-senti stuff

### PR DESCRIPTION
**Pull request by schan27**

### CODE REVIEW

File: openrice-senti/playground.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    if "yue-scratch" in str(model_path):
+        tokenizer_class = AlbertTokenizer
+    else:
+        tokenizer_class = BertTokenizer
+
+    # Initialize tokenizer
+    tokenizer = tokenizer_class.from_pretrained(model_path)
+
+    def tokenize_function(examples):
+        return tokenizer(examples["text"], padding=True, truncation=True, max_length=512)
```
Comment (Swithord) [inline openrice-senti/playground.py]: as I recall, some models (at least the transfer model) were trained with max_length=128. i'm not 100% sure about the transformer architecture, but fine-tuning with a context window of 512 here would yield poor performance for those models right? maybe we should train with max_length=128

File: openrice-senti/playground.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    model = BertForSequenceClassification.from_pretrained(
+        model_path,
+        num_labels=3,  # Binary classification
+    )
+
+    # Define training arguments
+    training_args = TrainingArguments(
+        output_dir=f"./results/{model_name}",
+        learning_rate=2e-5,
+        per_device_train_batch_size=4,
```
Comment (Swithord) [inline openrice-senti/playground.py:109]: i feel like the batch size could be larger (maybe 16?), since we have enough data. partly for stability, partly for consistency with our setup for other tasks

Comment (schan27) [reply openrice-senti/playground.py:109]: Sadly this is the max my mac can handle :( Do you think you might be able to try it if you have access to servers?

File: openrice-senti/playground.py (hunk 1 above)
Comment (Aatlantise) [reply openrice-senti/playground.py]: Are openrice sequences long enough to warrant context window of 512? I wouldn't know for sure but ideally pre-training and downstream max_len would be the same.

File: openrice-senti/playground.py (hunk 2 above)
Comment (Swithord) [reply openrice-senti/playground.py:109]: ahh fair, which mac are you using? batch_size=16 works fine for other tasks on my M1 Pro. i'll try to run it when i'm home

File: openrice-senti/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+## Description
+The files contain random reviews from the OpenRice Hong Kong section. 
+These are reviews of restaurants in Hong Kong. The dataset is filtered to contain
+only those written in Hong Kong Cantonese (Hongkongese).
+
+The files are tab-separated with sentiment in column one and review text in
+column two. The classes are:
+* smile
+* ok
+* cry
```
Comment (Aatlantise) [inline openrice-senti/README.md:13]: Great readme! Thanks for this.

File: openrice-senti/playground.py (hunk 2 above)
Comment (Aatlantise) [reply openrice-senti/playground.py:109]: The results are in line with what I expected--the reviews may (or may not) ask for multi-seed evaluation; when they do that's something we can fix at that time.

**Review by Aatlantise — APPROVED** [review]
Great work! Thanks @schan27 .

File: openrice-senti/playground.py (hunk 1 above)
Comment (schan27) [reply openrice-senti/playground.py]: Sounds good - thanks for catching it! I've changed it so that when/if we rerun the max_length will be updated.

File: openrice-senti/README.md (hunk 3 above)
Comment (schan27) [reply openrice-senti/README.md:13]: This is copied from the original repo :)

File: openrice-senti/playground.py (hunk 2 above)
Comment (schan27) [reply openrice-senti/playground.py:109]: I'm using M2 air! Maybe with the reduced length I'll be able to increase the batch size - will experiment next time I run.
